frontend/app/ui_components/date/date-range-picker-directive.js

```diff
@@ -29,10 +29,16 @@
 module.exports = function(TimezoneService, ConfigurationService,
                           I18n, $timeout) {
   var parseDate = TimezoneService.parseDate,
+      parseISODate = TimezoneService.parseISODate,
       formattedDate = function(date) {
         return TimezoneService.parseDate(date).format('L');
       },
       formattedISODate = TimezoneService.formattedISODate,
+      customDateFormat = 'YYYY-MM-DD',
+      datepickerFormat = 'yy-mm-dd',
```

Comment:
It's for jQuery ui datepicker date format. While moment.js uses 'YYYY' for datepicker it will be 'yy'.